<launch>
    <node pkg="pr2lite_arm_navigation" type="fake_linact_pos.py" name="fake_linact_pos" />
</launch>
